### ISSUE TITLE
Incorrect marker location on reload ("Swords")

### ISSUE THREAD

**Issue by wansco** [allenhwkim/angularjs-google-maps#423]
I'm noticing an issue where the map/marker almost always goes to the same (seemingly random?) location north of Dublin in the town of "Swords". It doesn't seem to be a zero or ub-initialized GPS location. It also doesn't seem to matter what I set the marker to initially.

I'm using map-lazy-load and am in a single page app. The first time I load the page, the marker is in the correct spot. If I change state (angular-ui) and then hit the back button, the map and marker is back in "Swords". Under the state change, the page isn't exactly refreshed, but the dom elements are swapped out. I'm wondering if this has anything to do with #419

Also this appears to be tied to Firefox as I have not noticed the behavior in Chrome. If I inspect the DOM, the marker tag has the correct position.

I guess this is as much of a question of the significance of that town as a default position on the map? Its got me baffled. Anyone else seen this behavior?

Here's roughly where it is:
https://www.google.com/maps/place/53%C2%B028'13.5%22N+6%C2%B013'45.7%22W/@53.4703548,-6.2301191,182m/data=!3m1!1e3!4m2!3m1!1s0x0:0x0

**Comment by allenhwkim** [comment]
Do you mean the marker is off-center?
Of marker itself is wrongly located?

Anyway it's not related to issue #419. Please reproduce your error with plunker, so that I can see and experience what your issue is.

**Comment by wansco** [comment]
The marker is displayed in the wrong place. For example, I set the marker to be in Los Angeles, and when this issue pops up, the map shows the marker in that town "Swords" near Dublin. It doesnt seem to matter what the initial marker location is. When the issue is present (only after a state transition or a "back"), it always ends up rendering in the same place. A full page refresh resolves the issue.

When I inspect the DOM, the marker position is correct.

I'm having trouble recreating this in plunker because it seems to be dependent on state transitions, and might be a factor of some nested directives, particularly ng-show.

I changed some of my ng-show's to ng-if's and that seems to have cleared it up (which is why I suspect it might have some loose relation to #419, although its likely unrelated as the memory leak seems to be a google maps issue).

I was hoping someone else might have seen this behavior, but for now it seems to be isolated to my particular application.
